bee-block/src/payload/milestone/option/mod.rs

```diff
@@ -123,21 +125,32 @@ impl MilestoneOptions {
 }
 
 #[inline]
-fn verify_unique_sorted<const VERIFY: bool>(milestone_options: &[MilestoneOption], _: &()) -> Result<(), Error> {
+fn verify_unique_sorted<const VERIFY: bool>(milestone_options: &[MilestoneOption]) -> Result<(), Error> {
     if VERIFY && !is_unique_sorted(milestone_options.iter().map(MilestoneOption::kind)) {
         Err(Error::MilestoneOptionsNotUniqueSorted)
     } else {
         Ok(())
     }
 }
 
+#[inline]
+fn verify_unique_sorted_packable<const VERIFY: bool>(
+    milestone_options: &[MilestoneOption],
+    _visitor: &ProtocolParameters,
```

Comment:
For what is this visitor field required?

Reply:
It's not required. This a callback for packable so it automatically received the visitor whether we use it or not as Packable can't know. I guess in the future we can add more flags to packable to tell it to forward visitor or no but at the moment that's the way it is :/